### PR TITLE
Own CI, Add ci/ci.sh script

### LINKED ISSUE #149: Switch to own CI
Travis CI is too slow

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
Closes #152, closes #149 

Travis CI script moved to ci folder and not used anymore.

We now have own Jenkins on https://public-ci.pushtorefresh.com, average build time 5 minutes, but I want to disable pre-dexing, so it should be faster

@nikitin-da I'll comment when PR will be ready to review and merge

### CODE REVIEW

**Comment by artem-zinnatullin** [comment]
Testing closing/opening PR

**Comment by artem-zinnatullin** [comment]
Hm, I don't know why, but Jenkins don't want to remember part of Job configuration after restart, looks like a bug

**Comment by nikitin-da** [comment]
Oh, do you remember, I've already met similar problem. And that time we talked about permissions for Jenkins service, to write on disk. But I don't remember if it was helpful =(

**Comment by artem-zinnatullin** [comment]
Jenkins has permissions to write on disk, he need to download repo and build it :)

Actually, only little part of Job configuration got reset after restart, I'll file a bug in Jenkins Jira

**Comment by artem-zinnatullin** [comment]
Btw! 4 minutes and 4 seconds, much faster than Travis CI

**Comment by nikitin-da** [comment]
Cool!
"artem-zinnatullin closed this 6 minutes ago
artem-zinnatullin reopened this 6 minutes ago" <- Like Ben?

**Comment by artem-zinnatullin** [comment]
LOL :+1:

**Comment by artem-zinnatullin** [comment]
retest this please

**Comment by artem-zinnatullin** [comment]
So, @nikitin-da now you can review it :)

File: build.gradle [hunk 1]
```diff
@@ -58,4 +58,17 @@ ext {
...
+
+// Option to disable Pre-Dexing on CI env
+project.ext.preDexLibs = !project.hasProperty('disablePreDex')
+
+subprojects {
+    project.plugins.whenPluginAdded { plugin ->
+        if ("com.android.build.gradle.AppPlugin".equals(plugin.class.name)) {
+            project.android.dexOptions.preDexLibraries = rootProject.ext.preDexLibs
+        } else if ("com.android.build.gradle.LibraryPlugin".equals(plugin.class.name)) {
+            project.android.dexOptions.preDexLibraries = rootProject.ext.preDexLibs
```
Comment (nikitin-da) [inline build.gradle]: What is the different? Can we use logical or instead double if?

Comment (artem-zinnatullin) [inline build.gradle]: Actually, I just picked this code from
http://tools.android.com/tech-docs/new-build-system/tips

:)

вт, 24 марта 2015, 0:56, Dmitrii Nikitin <notifications@github.com>:

In build.gradle

Comment (artem-zinnatullin) [inline build.gradle]: I'll fix it

**Comment by artem-zinnatullin** [comment]
@nikitin-da updated

**Comment by nikitin-da** [comment]
Started 4 min 1 sec ago
Took 3 min 55 sec
Super!

**Pull request merged**